src/EditorFeatures/Core/Implementation/FindReferences/FindReferencesCommandHandler.cs

```diff
@@ -51,12 +52,15 @@ internal void FindReferences(ITextSnapshot snapshot, int caretPosition)
                     var service = document.Project.LanguageServices.GetService<IFindReferencesService>();
                     if (service != null)
                     {
-                        if (!service.TryFindReferences(document, caretPosition, context))
+                        using (Logger.LogBlock(FunctionId.CommandHandler_FindAllReference, context.CancellationToken))
```

Comment:
One of the things we tried to do in the past was split the results based on what was being found: locals vs types, etc.  Also, we tried to include some extra information about how many references were found.  Is that interesting to do again?